.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# [Choice] Ruby version (use -bullseye variants on local arm64/Apple Silicon): 3, 3.1, 3.0, 2, 2.7, 2.6, 3-bullseye, 3.1-bullseye, 3.0-bullseye, 2-bullseye, 2.7-bullseye, 2.6-bullseye, 3-buster, 3.1-buster, 3.0-buster, 2-buster, 2.7-buster, 2.6-buster
+ARG VARIANT=3.1-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/ruby:0-${VARIANT}
+
+# Install Rails
+RUN gem install rails webdrivers 
```

Comment:
Not sure this is needed since you do

```
"postCreateCommand": "bundle install && ./bin/rails db:setup",
```

in `.devcontainer/devcontainer.json`

Reply:
True, likely not.  The Dockerfile in this case is autogenerated boilerplate from the default "Rails and Postgres" devcontainer, my guess is they install rails at the beginning such that the `rails` CLI binary is on the default path for people to be able to use to generate new projects.  I'll look at potentially removing it here.